### PR TITLE
[PyTorch] Add support for operator reshape

### PR DESCRIPTION
**Pull request by zrphercule**
Summary:

Documentation:

[Optional Fixes #issue]

Test Plan:

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 1]
```diff
@@ -445,6 +445,33 @@ void PyTorchModelLoader::loadMaxPool2d(const torch::jit::Node *ptNode) {
   addGlowNodeValue(outputs[0], output);
 }
 
+void PyTorchModelLoader::loadReshape(const torch::jit::Node *ptNode) {
```
Comment (rdzhabarov) [inline torch_glow/src/PyTorchModelLoader.cpp]: General question.

Could any of the load methods fail, like some of the params are not supported etc?
In that case I'm wondering why it's all `void`.
cc: @jackm321

Comment (zrphercule) [reply torch_glow/src/PyTorchModelLoader.cpp]: Good point, I think we need error checking mechanism.

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 2]
```diff
@@ -445,6 +445,33 @@ void PyTorchModelLoader::loadMaxPool2d(const torch::jit::Node *ptNode) {
...
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  assert(inputs.size() == 2);
+  assert(outputs.size() == 1);
+
+  // Indexes of aten::reshape inputs.
+  struct Inputs {
+	enum {
```
Comment (jackm321) [inline torch_glow/src/PyTorchModelLoader.cpp]: formatting looks weird here?

File: torch_glow/src/PyTorchModelLoader.cpp (hunk 1 above)
Comment (jackm321) [reply torch_glow/src/PyTorchModelLoader.cpp]: Yeah, we will add llvm::Errors like in c2 model loader. It's on the todo list :)

File: torch_glow/tests/utils.py [hunk 3]
```diff
@@ -31,3 +31,31 @@ def jitVsGlow_(f_torch, f_glow, *inputs):
     assert num_glow_nodes == 1, "Expected exactly 1 Glow node, found {}".format(num_glow_nodes)
 
     assert torch.allclose(torch_res, glow_res, atol=01e-6)
+
+def jitScriptVsGlow(f_torch, f_glow, *inputs):
```
Comment (jackm321) [inline torch_glow/tests/utils.py]: Why is this needed? Can you add comments to it?

File: torch_glow/src/PyTorchModelLoader.cpp (hunk 2 above)
Comment (jfix71) [reply torch_glow/src/PyTorchModelLoader.cpp]: Hm, I guess we're not running the clang-format CI check in here...?

Comment (jackm321) [reply torch_glow/src/PyTorchModelLoader.cpp]: Oh damn good catch, lets fix that

Comment (jackm321) [reply torch_glow/src/PyTorchModelLoader.cpp]: https://github.com/pytorch/glow/pull/3304
